### PR TITLE
refactor(locales): qb to ox

### PR DESCRIPTION
**Pull request by mafewtm**
## Description

Switches over to the ox locales instead of the old qb locales

## Checklist

- [x] I have personally loaded this code into an updated Qbox project and checked all of its functionality.
- [x] My pull request fits the contribution guidelines & code conventions.

### CODE REVIEW

File: client/main.lua [hunk 1]
```diff
@@ -5,6 +5,8 @@ local pedsSpawned = false
 local table_clone = table.clone
 local blips = {}
 
+lib.locale()
+
```
Comment (solareon) [inline client/main.lua]:
```suggestion
```
Use fxmanifest locale

File: server/main.lua [hunk 2]
```diff
@@ -1,6 +1,8 @@
 local sharedConfig = require 'config.shared'
 local JOBS = exports.qbx_core:GetJobs()
 
+lib.locale()
+
```
Comment (solareon) [inline server/main.lua]:
```suggestion
```
Use fxmanifest locale

File: fxmanifest.lua [hunk 3]
```diff
@@ -7,9 +7,6 @@ version '1.0.0'
```
Comment (solareon) [inline fxmanifest.lua:7]:
```suggestion

ox_lib 'locale'

```
Magic

**Review by solareon — CHANGES_REQUESTED** [review]

**Review by solareon — APPROVED** [review]

**Review by BerkieBb — APPROVED** [review]

**Comment by BerkieBb** [comment]
Fix lint issues

**Pull request merged**